windows-installer/auto-update.ps1

```diff
@@ -0,0 +1,134 @@
+[CmdletBinding()]
+param()
+begin {}
+process {
+    New-EventLog -Source "certbot/auto-update.ps1" -LogName "CertbotAutoUpdate" -ErrorAction SilentlyContinue
+
+    function Write-Message($message, $level = "Information") {
+        Write-EventLog -Source "certbot/auto-update.ps1" -LogName "CertbotAutoUpdate" -EventID 1 -EntryType $level -Message $message
+        Write-Host $message
+    }
+
+    function Write-Error($message) {
+        Write-EventLog -Source "certbot/auto-update.ps1" -LogName "CertbotAutoUpdate" -EventID 1 -EntryType Error -Message $message
```

Comment:
nit: Instead of defining `"certbot/auto-update.ps1"`, `"CertbotAutoUpdate"`, and maybe the event ID(?) in multiple places, what do you think about defining them earlier in the script as variables and reusing them here.

Quickly testing it, powershell seems to allow scoping like
```
$foo = "foo"
function Echo-Foo() {
    echo $foo
}
```

Reply:
Done.